### DISCUSSION TITLE
Where can administrators view the email address of comments left by guests?

### DISCUSSION THREAD

**Discussion by RiskeyL** [requarks/wiki#6429] [Help / Questions] [answered]
The doc says "The guest email address and IP address can only be seen by administrators with the manage:system permission.", but after I logged in as administrator (which I've doublechecked in the admin panel has the manage:system permission), I couldn't see the address either along with the comments or anywhere else in the admin panel. 

Did I miss anything?? Any help is very much appreciated. Thanks!!!

**Comment by NGPixel** [comment, marked as answer]
That info is currently not shown in the UI although it is stored in the DB. So you would need to look into the `comments` table in the DB for now.
